sql/messaging_system_derived/cfr_active_users_v1/query.sql

```diff
@@ -0,0 +1,96 @@
+-- This calculates the daily, weekly, monthly active clients based on the pings
+-- in `messaging_system_stable.cfr_v1`. A 7-day averaged daily is also calculated.
+-- The results are inserted into `messaging_system_derived.cfr_active_clients_v1`.
+-- Phase 1: DELETE the data for @submission_date if any.
+DELETE FROM
+  cfr_active_clients_v1
+WHERE
+  submission_date = @submission_date;
```

Comment:
This query contains multiple statements, which under the hood turns it into a BigQuery script rather than a simple query. We expect all incremental `query.sql` files to represent simple queries. Of note, you cannot set `destination_table` in a script.